app/src/androidTest/java/org/kiwix/kiwixmobile/intro/IntroRobot.kt

```diff
@@ -32,14 +32,12 @@ fun intro(func: IntroRobot.() -> Unit) = IntroRobot().applyWithViewHierarchyPrin
 class IntroRobot : BaseRobot() {
 
   private val getStarted = ViewId(R.id.get_started)
-  private val viewPager = ViewId(R.id.view_pager)
 
   fun swipeLeft() {
     isVisible(getStarted)
     isVisible(TextId(R.string.welcome_to_the_family))
     isVisible(TextId(R.string.humankind_knowledge))
     attempt(10) {
-      isVisible(viewPager).swipeLeft()
```

Comment:
Why are we removing this test?

Reply:
hi @gouri-panda ,
i have removed this because  ```ViewPager``` is automatically swipe after some interval and sometimes this code leads to ```ViewNotFound Execption```.